sites/friday/src/get-started/laravel/deploy/octane.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
This part of the guide isn't linked from the previous step in the guide (Blackfire). Do we need a button at the end of blackfire?

Reply:
Good catch. I added a link in the Blackfire section.